src/components/ui/button.tsx

```
@@ -0,0 +1,54 @@
+    },
+  }
+)
+
+export interface ButtonProps
+  extends React.ButtonHTMLAttributes<HTMLButtonElement>,
+    VariantProps<typeof buttonVariants> {
+  asChild?: boolean
+}
+
+const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
+  ({ className, variant, size, asChild = false, ...props }, ref) => {
```

ESLint

[error] 'asChild' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)